pkg/apis/core/types.go

```diff
@@ -635,6 +692,18 @@ type PersistentVolumeClaimStatus struct {
 	// +mapType=granular
 	// +optional
 	AllocatedResourceStatuses map[ResourceName]ClaimResourceStatus
+	// volumeAttributesClassName is the name of VolumeAttributesClass which is latest applied by
+	// the CSI driver. This field is not set by default but when volume modification is complete,
+	// it's set to the applied VolumeAttributesClass.
+	// This is an alpha field and requires enabling VolumeAttributesClass feature.
+	// +featureGate=VolumeAttributesClass
+	// +optional
+	VolumeAttributesClassName *string
+	// ModifyVolumeStatus represents the status object of ControllerModifyVolume operation
+	// This is an alpha field and requires enabling VolumeAttributesClass feature.
+	// +featureGate=VolumeAttributesClass
+	// +optional
+	ModifyVolumeStatus ModifyVolumeStatus
```

Comment:
This should be a pointer so we can clear it when there's no operation in progress.

Reply:
So we do not want it to be nil ever.

Reply:
It is nil when there is nothing to modify